src/c-writer.cc

```diff
@@ -428,6 +428,7 @@ static constexpr char kParamSuffix =
 static constexpr char kLabelSuffix = kParamSuffix + 1;
 
 static constexpr char kSymbolPrefix[] = "w2c_";
```

Comment:
Should we rename this `kGlobalSymbolPrefix`?

Reply:
Done in ce882d6a23803390ab098d908c1df2bac501651a.